testdata/resolvers/pipelineruns/bundles-resolver-pipelinerun-param.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: bundles-resolver-pipelinerun-param
+spec:
+  pipelineRef:
+    resolver: bundles
+    params:
+    - name: bundle
+      description: |
+        The image was created and pushed to quay.io using the 'tkn bundle push' command,
+        it includes 2 yaml files: bundles-resolver-task and bundles-resolver-pipeline.
```

Comment:
We should include these two files in this repository, then you can include paths to them (`testdata/.../x.yaml`) here